eventsources/common/webhook/webhook.go

```diff
@@ -99,15 +100,23 @@ func startServer(router Router, controller *Controller) {
 	handler := controller.ActiveServerHandlers[route.Context.Port]
 
 	routeName := route.Context.Port + route.Context.Endpoint
-
 	r := handler.GetRoute(routeName)
 	if r == nil {
 		r = handler.NewRoute().Name(routeName)
 		r = r.Path(route.Context.Endpoint)
 	}
-
 	r.HandlerFunc(router.HandleRoute)
 
+	healthCheckRouteName := route.Context.Port + "/health"
+	healthCheckRoute := handler.GetRoute(healthCheckRouteName)
+	if healthCheckRoute == nil {
+		healthCheckRoute = handler.NewRoute().Name(healthCheckRouteName)
+		healthCheckRoute = healthCheckRoute.Path("/health")
+	}
+	healthCheckRoute.HandlerFunc(func(writer http.ResponseWriter, request *http.Request) {
```

Comment:
can we add `HandlerFunc` under the if == nil condition?

Reply:
done.